05_syntax_lab/02.py

```diff
@@ -1,6 +1,9 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+sum = 0
+for i in range(8):
+    from random import randint
+    sum += randint(1, 101)
+print sum
+
+if sum % 7 == 0:
+    print "Boom"
+

```

Comment:
The import statement should be at the beginning of the file, right after the comment explaining what does the script do.
Please check how many numbers do you generate in this script.